lib/plots/pane.js

```diff
@@ -35,7 +35,9 @@ export default class PlotPane extends PaneItem {
       </div>,
       <Button icon='circle-slash' alt='Forget Plot' onclick={() => this.show()} />
     ];
-    if (this.item && this.item.toolbar) buttons.push(toView(this.item.toolbar))
+
+    if (this.item && this.item.toolbar) buttons = buttons.concat(toView(this.item.toolbar))
```

Comment:
Do you mean mapping `toView` over `item.toolbar`? Why would that be better than to let `toView` handle the "dispatch"?